### PR TITLE
/graph/tag fix lint errors/warnings

### PR DESCRIPTION
**Pull request by brahmaroutu**
Addresses #14756
Signed-off-by: Srini Brahmaroutu srbrahma@us.ibm.com

### CODE REVIEW

File: graph/tags/tags.go [hunk 1]
```diff
@@ -6,8 +6,10 @@ import (
 	"github.com/docker/distribution/registry/api/v2"
 )
 
+// DEFAULTTAG (latest) is used when tag is not specified.
 const DEFAULTTAG = "latest"
```
Comment (runcom) [inline graph/tags/tags.go]: this should be `DefaultTag`

`DefaultTag defines the default tag used when performing images related actions and no tag strings is specified`
maybe @stevvooe has a better comment for this tho

File: graph/tags/tags.go [hunk 2]
```diff
@@ -6,8 +6,10 @@ import (
...
 
+// ErrTagInvalidFormat is the error returned if tag is invalid.
```
Comment (runcom) [inline graph/tags/tags.go]: "ErrTagInvalidFormat is returned when a tag is invalid."

File: graph/tags/tags.go [hunk 3]
```diff
@@ -16,7 +18,7 @@ func (e ErrTagInvalidFormat) Error() string {
 	return fmt.Sprintf("Illegal tag name (%s): only [A-Za-z0-9_.-] are allowed ('.' and '-' are NOT allowed in the initial), minimum 1, maximum 128 in length", e.name)
 }
 
-// ValidateTagName validates the name of a tag
+// ValidateTagName validates the name of a tag.
```
Comment (runcom) [inline graph/tags/tags.go]: I'll add: "It returns an error if the given name is an emtpy string. If name does not match v2.TagNameAnchoredRegexp regexp, it returns ErrTagInvalidFormat"

Comment (brahmaroutu) [inline graph/tags/tags.go]: @runcom Please see the updated changes.

File: graph/tags/tags.go (hunk 1 above)
Comment (brahmaroutu) [inline graph/tags/tags.go]: I would keep it DEFAULTTAG, if not I will have to change the api package.

Comment (brahmaroutu) [inline graph/tags/tags.go]: I have renamed DEFAULTTAG to DefaultTag. Please review.

**Comment by stevvooe** [comment]
LGTM

**Comment by jfrazelle** [comment]
needs rebase

**Comment by tiborvass** [comment]
@brahmaroutu needs a rebase

**Comment by brahmaroutu** [comment]
@tiborvass I have rebased the code.

**Comment by tiborvass** [comment]
@brahmaroutu sorry needs another rebase -_-

**Comment by brahmaroutu** [comment]
@tiborvass Rebased.

**Comment by calavera** [comment]
LGTM. Merging!

**Pull request merged**